src/transformers/image_utils.py

```diff
@@ -66,6 +68,30 @@ class ChannelDimension(ExplicitEnum):
     LAST = "channels_last"
 
 
+class AnnotationFormat(ExplicitEnum):
+    COCO_DETECTION = "coco_detection"
+    COCO_PANOPTIC = "coco_panoptic"
+
+
+class DeprecatedEnumMeta(EnumMeta):
+    def __init__(cls, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        warnings.warn(
+            f"`{cls.__name__}` is deprecated and will be removed in a forthcoming version. "
```

Comment:
We try to be specific with a version number in deprecation warnings - typically two after the release it's added to (v4.36 in this case)

```suggestion
            f"`{cls.__name__}` is deprecated and will be removed in v4.38. "
```

Reply:
Makes sense - I will update the message.